### PR TITLE
[GAE-Java] Make GAE Java SDK version align with GraphScope

### PR DESCRIPTION
**Pull request by zhanglei1949**
## Related issue number

Make GAE Java SDK version not fixed in code/configuration. BUT we still need to change `analytical_engine/java/pom.xml` for GraphScope version change.

### CODE REVIEW

**Review by sighingnow — APPROVED** [review]

File: coordinator/gscoordinator/utils.py [hunk 1]
```diff
@@ -157,9 +164,8 @@
 LLVM4JNI_USER_OUT_DIR_BASE = "user-llvm4jni-output"
 PROCESSOR_MAIN_CLASS = "com.alibaba.graphscope.annotation.Main"
 JAVA_CODEGNE_OUTPUT_PREFIX = "gs-ffi"
-GRAPE_PROCESSOR_JAR = os.path.join(
-    GRAPHSCOPE_HOME, "lib", "grape-runtime-0.16.0-shaded.jar"
-)
+GRAPE_PROCESSOR_JAR = glob.glob(GRAPHSCOPE_HOME + "/lib/grape-runtime*.jar")[0]
```
Comment (siyuan0322) [inline coordinator/gscoordinator/utils.py:167]: This [0] seems dangerous...
Could you handle the exception incase the jar is not there? So we won't crash the coordinator.

File: coordinator/gscoordinator/utils.py [hunk 2]
```diff
@@ -121,14 +121,21 @@
...
 # ANALYTICAL_ENGINE_JAVA_HOME
 ANALYTICAL_ENGINE_JAVA_HOME = ANALYTICAL_ENGINE_HOME
-# ANALYTICAL_ENGINE_JAVA_INIT_CLASS_PATH=os.path.join(ANALYTICAL_ENGINE_JAVA_HOME, "lib/*")
-# There should be only grape-runtime.jar we need
-ANALYTICAL_ENGINE_JAVA_INIT_CLASS_PATH = (
-    ANALYTICAL_ENGINE_JAVA_HOME + "/lib/grape-runtime-0.16.0-shaded.jar"
+
+ANALYTICAL_ENGINE_JAVA_RUNTIME_JAR = glob.glob(
+    ANALYTICAL_ENGINE_JAVA_HOME + "/lib/grape-runtime*.jar"
+)[0]
```
Comment (siyuan0322) [inline coordinator/gscoordinator/utils.py:127]: And this,

File: coordinator/gscoordinator/utils.py [hunk 3]
```diff
@@ -121,14 +121,21 @@
...
+ANALYTICAL_ENGINE_JAVA_GRAPHX_JAR = glob.glob(
+    ANALYTICAL_ENGINE_JAVA_HOME + "/lib/graphx-on-graphscope*.jar"
+)[0]
```
Comment (siyuan0322) [inline coordinator/gscoordinator/utils.py:130]: This

**Review by siyuan0322 — APPROVED** [review]

File: coordinator/gscoordinator/utils.py (hunk 1 above)
Comment (zhanglei1949) [reply coordinator/gscoordinator/utils.py]: Thanks for the suggestion, I have changed to 
```python 
ANALYTICAL_ENGINE_JAVA_RUNTIME_JAR = os.path.join(
    ANALYTICAL_ENGINE_JAVA_HOME,
    "lib",
    "grape-runtime-{}-shaded.jar".format(__version__),
)
```
where the `__version__` is imported from version.py

**Review by lidongze0629 — APPROVED** [review]

**Pull request merged**